stable/aws-for-fluent-bit/values.yaml

```diff
@@ -141,6 +164,74 @@ elasticsearch:
   # extraOutputs: |
   #   Index = my-index
 
+s3:
+  enabled: true
+  match: "*"
+  bucket:
+  region: "us-east-1"
+  jsonDateKey: "date"
+  jsonDateFormat: "iso8601"
+  totalFileSize: "100M"
+  uploadChunkSize: 50M
+  uploadTimeout: "10m"
+  storeDir: "/tmp/fluent-bit/s3"
+  storeDirLimitSize: 0
+  s3KeyFormat: /fluent-bit-logs/$TAG/%Y/%m/%d/%H/%M/%S
```

Comment:
same comment as on CW, I don't think the name of the tool should be in the pod log group/key name. How about- `pod-logs`?

Reply:
Done